pkg/coscheduling/coscheduling.go

```diff
@@ -132,11 +133,38 @@ func (cs *Coscheduling) Less(podInfo1, podInfo2 *framework.QueuedPodInfo) bool {
 func (cs *Coscheduling) PreFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod) *framework.Status {
 	if err := cs.pgMgr.PreFilter(ctx, pod); err != nil {
 		klog.Error(err)
+		state.Write(cs.getStateKey(pod), NewNilStateData())
 		return framework.NewStatus(framework.Unschedulable, err.Error())
 	}
 	return framework.NewStatus(framework.Success, "")
 }
 
+// PostFilter is use for rejecting the group pods if a pod does not pass PreFilter or Filter.
+func (cs *Coscheduling) PostFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod,
+	filteredNodeStatusMap framework.NodeToStatusMap) (*framework.PostFilterResult, *framework.Status) {
+	// check if failed when pre filter
+	_, err := state.Read(cs.getStateKey(pod))
+	if err == nil {
+		state.Delete(cs.getStateKey(pod))
+		return &framework.PostFilterResult{}, framework.NewStatus(framework.Unschedulable, "")
```

Comment:
`framework.NewStatus(framework.Unschedulable)`, or provide the specific error reason `framework.NewStatus(framework.Unschedulable, ...)`